src/stores/business.ts

```diff
@@ -271,59 +269,54 @@ export const useBcrosBusiness = defineStore('bcros/business', () => {
   const isAllowed = (action: AllowableActionE): boolean => {
     const isBusiness = !!currentBusiness.value?.identifier
 
+    const { isStaffAccount } = useBcrosAccount()
```

Comment:
This should be a check for an authorized action.

Actually, GitHub Actions says this is never used, so I guess you can delete it.